CHANGELOG.md

```diff
@@ -91,6 +91,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Fix array hashCode calculation in ResyncReplicationRequest ([#16378](https://github.com/opensearch-project/OpenSearch/pull/16378))
 - Fix missing fields in task index mapping to ensure proper task result storage ([#16201](https://github.com/opensearch-project/OpenSearch/pull/16201))
 - Fix typo super->sb in method toString() of RemoteStoreNodeAttribute ([#15362](https://github.com/opensearch-project/OpenSearch/pull/15362))
+- [Workload Management] query group create/update APIs were not properly setup ([16422](https://github.com/opensearch-project/OpenSearch/pull/16422))
```

Comment:
Can we update the changelog to say "Extending QueryGroup create/updation TransportActions from *ClusterManagerNode* classes"?

Reply:
I see you have already done it! Thanks!!